content/topics/How-to-ask-for-help-with-your-code/_index.md

```diff
@@ -9,16 +9,24 @@ Practice asking good questions. It's like a superpower. If you are easy to help
 
 ### These also include:
 
-#### how to ask good questions on slack:
+#### Where to ask good questions
 
-* all Slack questions should be addressed on the slack **Tech Assist** channel.
- - This helps you get to a broad audience from the tech department (.e.g Recruits, Managers and Alumni).
- - Please don't send screenshots what you can do is use backtick (` `) characters to represent a code form.
- - Write full error messages. (Summarise if you can and make sure you understand the error before sending it)
+ - Slack
```

Comment:
Talk about slack here:
- please use the #tech-assist channel. If you don't get help there then blah blah other places